lib/constants.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/material.dart';
+
+const kTextColor = Color(0xFF535353);
```

Comment:
I don't recommend using custom prefixes/hungarian notation/etc. in Dart. Historically, it was invented to manage complexity on the projects, but right now the direction industry moves towards is "don't create complex code, and you won't need custom prefixes for your variables)